packages/engine/Source/Scene/Megatexture.js

```diff
@@ -33,52 +31,33 @@ function Megatexture(
   dimensions,
   channelCount,
   componentType,
-  availableTextureMemoryBytes,
+  availableTextureMemoryBytes = 134217728,
```

Comment:
Perhaps worth keeping the explanation of the default from below:

```suggestion
  availableTextureMemoryBytes = 134217728, // 1024x1024 @ 128bpp
```